cycode/cyclient/scan_client.py

```diff
@@ -42,6 +43,8 @@ def zipped_file_scan(self, scan_type: str, zip_file: InMemoryZip, scan_id: str,
 
     def zipped_file_scan_async(self, zip_file: InMemoryZip, scan_type: str, scan_parameters: dict,
                                is_git_diff: bool = False) -> models.ScanInitializationResponse:
+        if scan_type == SECRET_SCAN_TYPE:
+            scan_type = "secrets" #the scanService endpoint is secrets and not secret
```

Comment:
extract to method - get_scan_uri which will return {scan_type}/repository.
if secret
   return secrets/repository
else
   {scan_type}/repository
anyway, avoid overriding method input and I think that it will be clear so you can skip the comment.